diff --git a/node_modules/jest-runner/build/index.d.ts b/node_modules/jest-runner/build/index.d.ts
index c71e8fb..289a96b 100644
--- a/node_modules/jest-runner/build/index.d.ts
+++ b/node_modules/jest-runner/build/index.d.ts
@@ -20,6 +20,8 @@ declare class TestRunner {
     private _context;
     readonly isSerial?: boolean;
     constructor(globalConfig: Config.GlobalConfig, context?: JestTestRunnerContext);
+    get testNamePattern(): string | undefined;
+    set testNamePattern(pattern: string | undefined);
     runTests(tests: Array<JestTest>, watcher: JestTestWatcher, onStart: JestOnTestStart, onResult: JestOnTestSuccess, onFailure: JestOnTestFailure, options: JestTestRunnerOptions): Promise<void>;
     private _createInBandTestRun;
     private _createParallelTestRun;
diff --git a/node_modules/jest-runner/build/index.js b/node_modules/jest-runner/build/index.js
index cc6e918..0217afb 100644
--- a/node_modules/jest-runner/build/index.js
+++ b/node_modules/jest-runner/build/index.js
@@ -75,6 +75,14 @@ class TestRunner {
     this._context = context || {};
   }
 
+  get testNamePattern() {
+    return this._globalConfig.testNamePattern;
+  }
+
+  set testNamePattern(pattern) {
+    this._globalConfig = {...this._globalConfig, testNamePattern: pattern};
+  }
+
   async runTests(tests, watcher, onStart, onResult, onFailure, options) {
     return await (options.serial
       ? this._createInBandTestRun(tests, watcher, onStart, onResult, onFailure)
